config/routes.rb

```diff
@@ -5,6 +5,8 @@
     resources :tasks
   end
 
+  delete "/logout", to: "sessions#destroy", as: "logout"
+
   resources :users, only: %i[new create]
-  resource :sessions, only: %i[new create show]
+  resource :sessions, only: %i[new create show destroy]
```

Comment:
```suggestion
  resource :sessions, only: %i[new create show]
```
You doesn't need in this action because you specify `sessions#destroy` above